docs/source/sysadmin_guide/pbft_adding_removing_node.rst

```diff
@@ -0,0 +1,158 @@
+******************************
+Adding or Removing a PBFT Node
+******************************
+
+Membership of a Sawtooth PBFT network is controlled by the on-chain setting
+``sawtooth.consensus.pbft.members``. An administrator must update this setting
+when :ref:`adding a new node <adding-a-pbft-node-label>` or
+:ref:`removing an existing node <removing-a-pbft-node-label>`.
+
+All nodes in the network monitor this setting. When a node detects a change,
+it updates its local list of PBFT members to match the new list.
+
+.. _adding-a-pbft-node-label:
+
+Adding a PBFT Node
+==================
+
+To add a new node to an existing PBFT network, you will install and configure
+the node, start it and wait for it to catch up with the rest of the network,
```

Comment:
Is this where you would add the new node as a `--peer` to the existing nodes? I wonder if we should cover that part of the process too.

Reply:
The other procedure describes how to configure a node as a peer on a Sawtooth network. (At least, it should - if it doesn't, that's a problem with that other procedure.)  I'd prefer to keep all the generic info in the other procedure so that this one can be focused on the PBFT-only information.

Reply:
I checked - the peering information is documented in this topic (see step 4):
https://sawtooth.hyperledger.org/docs/core/nightly/master/sysadmin_guide/off_chain_settings.html

I know that a PBFT network must be "fully peered".  @ltseeley , does that mean that the admin(s) must modify the off-chain peers settings on all nodes in the network?  Or is it sufficient to list all other nodes for this new node?

In any case (and to @rberg2 's comment), does this procedure need a note to explain how to handle peering on the new node?

Also, when removing a node, does the off-chain peering setting need to be changed on the remaining nodes when a node is removed from the network?

Reply:
It's sufficient to list all other nodes ONLY for the new node. Connections between validators are bi-directional.

Reply:
I shall add info in step 1 to mention that PBFT requires full peering and to link to the procedure that describes how to change the off-chain config file. Thanks, @rberg2 for this helpful comment!

Reply:
I shall also add a note in the "remove a PBFT node" procedure that you do NOT have to delete the removed node from the list of peers on other node. (You can do so if you want, but it's not required.)